src/data/modifiers.txt

```diff
@@ -5013,6 +5018,7 @@ Item	ticksilver spurs	Adventures: +5
 
 # Campground equipment section of modifiers.txt
 
+Item	A Guide to Burning Leaves	Leaves: +0.5
```

Comment:
duplicate modifiers for item (with free pull below)

Reply:
Hm. Will it be fine to merge these?